docs/maintenance/styleguide.md

```diff
@@ -49,9 +49,156 @@ Order buttons correctly:
 - ❌ Cancel (negative) | Save (positive)
 - ✅ Delete (negative) | ... (space) | Save (standard) | Publish (primary)
 
-### Captilization
+### Capitalization
 
 **Form labels**:
 
 - ✅ Resource type
 - ❌ Resource Type
+
+
+## Development practices
```

Comment:
I think we should separate JS/CSS best practices from the UX practice.

I would put this page under "Develop > Topic guides > CSS and JS best practices"